content/_data/changelogs/lts.yml

```diff
@@ -4389,6 +4389,331 @@
+
+  lts_changes: # compared to lts_predecessor 2.249.3 (selected by personal review)
+
+  - type: major bug
+    category: major bug
+    authors:
+    - tam116
+    references:
+      - issue: 64212
+      - url: "https://github.com/jenkinsci/packaging/pull/198"
+        title: pull 198
+    message: |-
+      Fix Debian / Ubuntu Java version check for Java 11.0.9.1.
+
+  - type: major rfe
+    category: major rfe
+    pull: 4909
+    references:
+      - pull: 4909
+      - url: https://github.com/jenkinsci/jep/blob/master/jep/224/README.adoc
+        title: JEP-224
+    authors:
+    - timja
+    message: |-
+      Graduate Overall/SystemRead permission to general availability (GA) status.
+
+  - type: major rfe
+    category: rfe
+    pull: 4951
+    authors:
+    - daniel-beck
+    message: >
+      SSHD module 2.7: Remove deprecated key exchange and MAC algorithms.
+    references:
+      - pull: 4951
+      - url: "https://github.com/jenkinsci/sshd-module/blob/master/CHANGELOG.md#27"
+        title: SSHD module 2.7 changelog
+
+  - type: rfe
+    category: rfe
+    pull: 4951
+    authors:
+    - daniel-beck
+    message: >
+      SSHD module 2.7: Allow configuring disabled key exchange and MAC algorithms through system properties.
+    references:
+      - pull: 4951
+      - url: "https://github.com/jenkinsci/sshd-module/blob/master/CHANGELOG.md#27"
+        title: SSHD module 2.7 changelog
+
+  - type: rfe
+    category: rfe
+    pull: 4926
+    issue: 63592
+    authors:
+    - fqueiruga
+    message: |-
+      Improve the layout and clarity of the page displayed when jobs are not yet created.
+
+  - type: rfe
+    category: rfe
+    pull: 4920
+    issue: 60972
+    authors:
+    - romenrg
+    message: |-
+      Name the first breadcrumb "Dashboard" for clarity.
+
+  - type: rfe
+    category: rfe
+    pull: 5016
+    authors:
+    - daniel-beck
+    message: |-
+      Remove TFS Plugin suggestion from Setup Wizard.
+
+  - type: rfe
+    category: rfe
+    pull: 4952
+    authors:
+    - daniel-beck
+    message: |-
+      Disable autocomplete of username on login form.
+
+  - type: rfe
+    category: rfe
+    pull: 4950
+    issue: 63795
+    authors:
+    - aHenryJard
+    message: |-
+      Allow users with the <code>Jenkins.MANAGE</code> permission to restart and safe restart Jenkins.
+
+  - type: rfe
+    category: rfe
+    pull: 4905
+    authors:
+    - tszmytka
+    message: |-
+      Do not render parts of a serialized <code>ConsoleNote</code> in truncated log output.
+
+  - type: rfe
+    category: rfe
+    pull: 4910
+    authors:
+    - daniel-beck
+    references:
+      - pull: 4910
+      - url: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy"
+        title: Cross-Origin-Opener-Policy at developer.mozilla.org
+    message: |-
+      Set <code>Cross-Origin-Opener-Policy</code> to <code>same-origin</code>.
```

Comment:
borderline too minor except for the likelihood to cause problems

Reply:
Intentionally retained because I assume some users will report issues related to it.  Having an entry in the changelog "feels better" to me when someone reports an issue with something we intentionally changed for their long term benefit.